src/components/SelectList/SelectList.tsx

```diff
@@ -268,9 +269,12 @@ const SelectList: FC<SelectListProps> = props => {
     const { widthProps, restProps } = extractWidthProps(withoutMargin);
     const { components, isDisabled, variant, inverted, size, error, label } = restProps;
 
+    const id = useGeneratedId(null);
```

Comment:
can it be without `null`?

```suggestion
    const id = useGeneratedId();
```

Reply:
If not, let's change the parameter of `useGeneratedId` to optional

Reply:
Perfect, was thinking the same. Done!